.github/workflows/ibis-backends.yml

```diff
@@ -243,12 +237,32 @@ jobs:
               extras:
                 - dask
           - os: ubuntu-latest
-            python-version: "3.9"
+            python-version: "3.11"
```

Comment:
shouldn't this be the 3.10, if it was 3.9 before?

Reply:
The diff is a bit weird. It was a Dask entry before, this one is a Flink entry. The reason we test on 3.11 for Flink still is that PyFlink doesn't support 3.12.